File: DairoNPC-Android/app/src/main/java/cn/dairo/npc/bean/NpcSetting.kt
package cn.dairo.npc.bean

data class NpcSetting (

    /**
     * 服务器
     */
    val host: String = "",

    /**
     * tcp端口
     */
    val tcpPort: String = "",

    /**
     * udp端口
     */
    val udpPort: String = "",

    /**
     * 秘钥
     */
    var key: String = "",
)

File: DairoNPC-IOS/DairoNPC/Bean/NpcSetting.swift
struct NpcSetting  : Codable{
    
    /**
     * 服务器
     */
    var host: String = ""
    
    /**
     * tcp端口
     */
    var tcpPort: String = ""
    
    /**
     * udp端口
     */
    var udpPort: String = ""
    
    /**
     * 秘钥
     */
    var key: String = ""
    
//    static var new12: NpcSetting{
//        get{
//            NpcSetting(host:"",tcpPort:"",udpPort:"",key:"")
//        }
//    }
}
